include/zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h

```diff
@@ -105,89 +102,107 @@ struct img_mgmt_upload_action {
 };
 
 /**
- * @brief Registers the image management command handler group.
+ * @brief Register the image management command handler group.
  */
 void img_mgmt_register_group(void);
 
 /**
- * @brief Unregisters the image management command handler group.
+ * @brief Unregister the image management command handler group.
  */
 void img_mgmt_unregister_group(void);
 
 /*
- * @brief Read info of an image give the slot number
+ * @brief Read info of an image at the specified slot number
+ *
+ * @param image_slot	 image slot number;
+ * @param ver		 output buffer for image version;
+ * @param hash		 output buffer for image hash;
+ * @param flags		 output buffer for image flags,
```

Comment:
remove ; and , from end of lines, this is not in line with zephyr documentation

Reply:
Done.